CONTRIBUTING.md

```diff
@@ -0,0 +1,298 @@
+   ```
+
+This command builds only the database. Other available targets for the ``scons`` command are:
+- ``mongod``
+- ``mongos``
+- ``mongo``
+- ``core`` (includes ``mongod``, ``mongos``, ``mongo``)
+- ``all``
+
+The built binaries are in the ``percona-server-mongodb`` directory. 
+
+#### Red Hat Enterprise Linux/CentOS
+
+1. Clone this repository and the AWS Software Development Kit for C++  repository
+```sh
+git clone https://github.com/percona/percona-server-mongodb.git
+git clone https://github.com/aws/aws-sdk-cpp.git
+```
+
+2. Install the dependencies for your operating system. The following command installs the dependencies for Centos 7:
+```sh
+sudo yum -y install centos-release-scl epel-release
+sudo yum -y install python-pip python-devel libpcap-devel gcc make cmake wget cmake3 gcc-c++ openssl-devel cyrus-sasl-devel snappy-devel zlib-devel bzip2-devel scons rpmlint rpm-build libopcodes libcurl-devel e2fsprogs-devel expat-devel lz4-devel openldap-devel krb5-devel devtoolset-8-gcc devtoolset-8-gcc-c++
+```
+
+3. Install Python dependencies
+   - Install Pip
+   ```sh
+   wget https://bootstrap.pypa.io/pip/2.7/get-pip.py
+   sudo python2 get-pip.py
+   ```
+   - Switch to the Percona Server for MongoDB branch that you are building 
+   ```sh
+   cd percona-server-mongodb && git checkout v3.6
+   ```
+   - Install Python modules
+   pip install --user setuptools --upgrade
+   pip install --user typing pyyaml regex Cheetah Cheetah3
+   pip install --user -r buildscripts/requirements.txt
+   ```
+4. Define Percona Server for MongoDB version (3.6.23 for the time of writing this document)
+```sh
+echo '{"version": "3.6.23"}' > version.json
+```
+
+5. Build a specific `curl` version
+   - Return to the home directory
+   ```sh
+   cd /
+   ```
+   - Fetch the package archive
+   ```sh
+   wget https://curl.se/download/curl-7.66.0.tar.gz
+   ```
+   - Unzip the package
+   ```sh
+   tar -xvzf curl-7.66.0.tar.gz && cd curl-7.66.0
+   ```
+   - Configure and build the package
+   ```sh
+   ./configure
+   sudo make install
+   ```
+
+6. Build the AWS Software Development Kit for C++ library
+   - Create a directory to store the AWS library
+   ```sh
+   mkdir -p /tmp/lib/aws
+   ```
+   - Declare an environment variable ``AWS_LIBS``  for this directory
+   ```sh
+   export AWS_LIBS=/tmp/lib/aws 
+   ``` 
+
+   - Percona Server for MongoDB is built with AWS SDK CPP 1.8.56 version. Switch to this version
+   ```sh
+   cd aws-sdk-cpp && git checkout 1.8.56
+   ```
+   - It is recommended to keep build files outside of the SDK directory. Create a build directory and navigate to it
+   ```sh
+   mkdir build && cd build
+   ```
+   - Generate build files using ``cmake``
+   ```sh
+   cmake3 .. -DCMAKE_C_COMPILER=/opt/rh/devtoolset-8/root/usr/bin/gcc  -DCMAKE_CXX_COMPILER=/opt/rh/devtoolset-8/root/usr/bin/g++ -DCMAKE_BUILD_TYPE=Release -DBUILD_ONLY="s3;transfer" -DBUILD_SHARED_LIBS=OFF -DMINIMIZE_SIZE=ON -DCMAKE_INSTALL_PREFIX="${AWS_LIBS}"
```

Comment:
typo: cmake3 -> cmake

Reply:
nope, it's not a type, it's required for centos7, otherwise error appears
CMake 3.1 or higher is required.  You are running version 2.8.12.2

for centos8 - cmake version is already 3.11.4 + for centos8 it's not required to define DCMAKE_C_COMPILER and DCMAKE_CXX_COMPILER